server/service/integration_mdm_test.go

```diff
@@ -277,15 +290,14 @@ func (s *integrationMDMTestSuite) mockDEPResponse(handler http.Handler) {
 	})
 }
 
-func (s *integrationMDMTestSuite) awaitTriggerProfileSchedule(t *testing.T, additionalWait time.Duration) {
-	ch := make(chan struct{})
-	s.onProfileScheduleDone = func() {
-		close(ch)
-	}
+func (s *integrationMDMTestSuite) awaitTriggerProfileSchedule(t *testing.T) {
+	// two schedules concurrently running: macOS and Windows
```

Comment:
> two schedules concurrently running: macOS and Windows

Should this be "two jobs running sequentially (macOS then Windows) on the same schedule"?

Reply:
you're right, thank you!